CMakeLists.txt

```diff
@@ -267,14 +267,18 @@ set(Python_FIND_UNVERSIONED_NAMES FIRST)
 
 include(CMake/DownloadFile.cmake)
 
+set(DOWNLOAD_FILES FALSE CACHE BOOL "Allow downloading files during the build")
```

Comment:
Could this be replaced with a narrower scope "USE_SYSTEM_SDL2_HEADERS"? Then, no changes would have to be made to WIN32, DARWIN, or 32-bit code paths

Reply:
Probably not a bad idea.

Reply:
Replaced with DOWNLOAD_SDL2, defaulting to TRUE